sql/moz-fx-data-shared-prod/firefox_accounts/nonprod_fxa_all_events/view.sql

```diff
@@ -0,0 +1,158 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.firefox_accounts.nonprod_fxa_all_events`
+AS
+-- Existing fxa event tables (stage)
+WITH auth_events AS (
+  SELECT
+    `timestamp`,
+    receiveTimestamp,
+    jsonPayload.fields.user_id,
+    jsonPayload.fields.country,
+    jsonPayload.fields.language,
+    jsonPayload.fields.app_version,
+    jsonPayload.fields.os_name,
+    jsonPayload.fields.os_version,
+    jsonPayload.fields.event_type,
+    jsonPayload.logger,
+    jsonPayload.fields.user_properties,
+    jsonPayload.fields.event_properties,
+    jsonPayload.fields.device_id,
+    "auth" AS fxa_log
+  FROM
+    `moz-fx-data-shared-prod.firefox_accounts_derived.nonprod_fxa_auth_events_v1`
+  WHERE
+    DATE(`timestamp`) < "2023-05-26"
+),
+content_events AS (
+  SELECT
+    `timestamp`,
+    receiveTimestamp,
+    jsonPayload.fields.user_id,
+    CAST(NULL AS STRING) AS country,
+    jsonPayload.fields.language,
+    jsonPayload.fields.app_version,
+    CAST(NULL AS STRING) AS os_name,
+    CAST(NULL AS STRING) AS os_version,
+    jsonPayload.fields.event_type,
+    jsonPayload.logger,
+    jsonPayload.fields.user_properties,
+    jsonPayload.fields.event_properties,
+    CAST(NULL AS STRING) AS device_id,
+    "content" AS fxa_log
+  FROM
+    `moz-fx-data-shared-prod.firefox_accounts_derived.nonprod_fxa_content_events_v1`
+  WHERE
+    DATE(`timestamp`) < "2023-05-26"
+),
+stdout_events AS (
+  SELECT
+    `timestamp`,
+    receiveTimestamp,
+    jsonPayload.fields.user_id,
+    CAST(NULL AS STRING) AS country,
+    jsonPayload.fields.language,
+    jsonPayload.fields.app_version,
+    jsonPayload.fields.os_name,
+    jsonPayload.fields.os_version,
+    jsonPayload.fields.event_type,
+    jsonPayload.logger,
+    jsonPayload.fields.user_properties,
+    jsonPayload.fields.event_properties,
+    jsonPayload.fields.device_id,
+    "payments" AS fxa_log
```

Comment:
We could rename the table, but I also think that it's not really necessary. I would update the fxa_log value to be `payments` and a note in the `metadata.yaml` to indicate this. As far as anything downstream from the view is concerned, it should rely on the `fxa_log` field (or whatever we decide to call it).

Reply:
I think calling it `fxa_service` would be a bit more intuitive/user friendly.

Reply:
I'm not necessarily opposed to this change. But I'm a bit worried that it will end up getting mixed up with this field: `jsonPayload.fields.event_properties.service` which seems to correspond to a Mozilla product.

Perhaps an alternative here would be something like `fxa_server` to avoid the confusion with mozilla services?

I do think it might be worth improving upon our naming, however, I'd suggest we treat it as a separate unit of work from the migration.

We could have a follow-up change/improvement to rename these fields since that would have to be done on the view level and would have downstream impact.

What are your thoughts @sean-rose ?

Reply:
Sure, `fxa_server` seems like a reasonable name, especially since in the new ETL the value is being extracted from `logger` values like "fxa-auth-server".

And I'm fine with punting on the `fxa_log` column renaming and doing it later.  However, in the same spirit, maybe this should be `"stdout" AS fxa_log` to keep with the current convention for now, and only change it to `"payments"` at the same time the `fxa_log` column renaming is done, since changing the value also potentially has downstream impacts?